src/eth.ts

```diff
@@ -297,7 +299,7 @@ export async function getBalance(
  */
 export function _createProvider(options: CallOptions = {}) : Provider {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  let provider: any = options.provider || (options.network || 'mainnet');
+  let provider: any = options.provider || (options.network || 'rsk-testnet');
```

Comment:
```suggestion
  let provider: any = options.provider || (options.network || 'testnet');
```
Use same name as in constants